test/Sentry.AspNetCore.Tests/WebIntegrationTests.PreFlightIgnoresTransaction.verified.txt

```diff
@@ -23,8 +23,8 @@
   Payloads: [
     {
       Source: {
-        Name: Unknown Route,
-        NameSource: Route,
+        Name: OPTIONS /Target,
+        NameSource: Url,
```

Comment:
Note, that this essentially would handle #1835, except that we've decided to no longer sent pre-flight requests.  The work to remove such transactions will come separately by @SimonCropp.